conduct.md

```diff
@@ -172,14 +172,16 @@ It is inevitable that conflicts will arise when working as a large group. It is
     compromise between class members involved.
     * Project managers may reassign different tasks to group members to solve a conflict.
     * The course instructor has a final say in how severe conflicts should be resolved.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for the conflicts.
+    * Establish a clear line of communication with other members before conflicts are brought to technical leaders or the instructor
+    * Maintain a professional working environment between colleagues when collaborating for projects.
```

Comment:
Check out line 168 to line 174. I know that your group did not contribute to this part, but since it's under the section of conflicts, I think it would be best for your groups to address some of the issues here. Most of the bullet points here are not started with a verb. Some entries are also either redundant or repetitive. And remember, this is a conduct for team members so it would be unnecessary to come up instructions for project managers and technical leaders, etc.